src/streaming-services/sync-template/SyncTemplateApi.ts

```diff
@@ -13,11 +13,7 @@ class _SyncTemplateApi extends Api {
 	}
 
 	// This method should load the next page of history items and update the sync store for the service.
-	loadHistory = async (
-		itemsToLoad: number,
-		lastSync: number,
-		lastSyncId: string
-	): Promise<void> => {
+	async loadHistory(itemsToLoad: number, lastSync: number, lastSyncId: string): Promise<void> {
 		// The code could look like this.
 		const store = getSyncStore('sync-template');
```

Comment:
On #75 I moved this logic to the `Api` class to make it more generic. But yeah, I'll try to work on something so that we only have to enter the ID once.